mangle-shebangs.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 
 SYSTEM_RELEASE_PATH=/etc/system-release
+RHEL8_REGEX="Red Hat Enterprise Linux release 8"
+FEDORA_REGEX="Fedora release"
 
-if [ -f $SYSTEM_RELEASE_PATH ] && [[ "$(cat $SYSTEM_RELEASE_PATH)" =~ "Red Hat Enterprise Linux release 8" ]]; then
-    # Replace the first line in .py to "#!/usr/bin/env python2" no matter what it was before
+# RHEL8 and Fedora30+ both treat shebangs of the form "#!/usr/bin/env
+# python" as errors
```

Comment:
The line split here is a bit awkward, is the comment too long for one line?

Reply:
+1, should in the same line.

Reply:
See commit cdcb0fd.

Reply:
@patelia, I rebased and force pushed.

Reply:
Amazing!